engines/wintermute/base/gfx/opengl/base_render_opengl3d.h

```diff
@@ -174,14 +154,17 @@ class BaseRenderOpenGL3D : public BaseRenderer {
 		return STATUS_OK;
 	};
 
-	bool drawSprite(BaseSurfaceOpenGL3D &tex, const Rect32 &rect, float zoomX, float zoomY, const Vector2 &pos,
-	                uint32 color, bool alphaDisable, Graphics::TSpriteBlendMode blendMode, bool mirrorX, bool mirrorY);
 	bool drawSpriteEx(BaseSurfaceOpenGL3D &tex, const Rect32 &rect, const Vector2 &pos, const Vector2 &rot, const Vector2 &scale,
-	                  float angle, uint32 color, bool alphaDisable, Graphics::TSpriteBlendMode blendMode, bool mirrorX, bool mirrorY);
+					  float angle, uint32 color, bool alphaDisable, Graphics::TSpriteBlendMode blendMode, bool mirrorX, bool mirrorY) override;
+
+	void renderSceneGeometry(BaseArray<AdWalkplane *> &planes, BaseArray<AdBlock *> &blocks, BaseArray<AdGeneric *> &generics, Camera3D *camera) override;
```

Comment:
Do all of these BaseArrays have to be BaseArrays?

I assume that they could at least be const-references?

Reply:
They are const now. Changing them to Common::Array would also affect the member variables of AdSceneGeometry, might be done later though.